executor/executor.go

```diff
@@ -1569,6 +1569,9 @@ func init() {
 		if err != nil {
 			return nil, err
 		}
+		if pi, ok := sctx.(processinfoSetter); ok {
+			pi.UpdateProcessInfo()
```

Comment:
We could comment on why the `UpdateProcessInfo` should be called here.

Reply:
done.